wfsim/load_resource.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, config=None):
         elif config['detector'] == "XENON1T":
             url_base = f'https://raw.githubusercontent.com/XENONnT/strax_auxiliary_files/{commit}/sim_files'
         elif config['detector'] == "XENONnT":
-            url_base = f'https://raw.githubusercontent.com/XENONnT/WFSim/{commit}/files'
+            url_base = f'https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/{commit}/sim_files'
```

Comment:
@mzks , this will never work as it is private

Reply:
I just copied TPC one. How should I modify them?

Reply:
I think `url_base = f'https://raw.githubusercontent.com/XENONnT/WFSim/{commit}/files'` (as it was) is of limited use but more useful than the `f'https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/{commit}/sim_files'` which certainly won't work.